cbt_enable_and_snapshot.py

```diff
@@ -46,6 +54,7 @@ def main():
                                 "CBT example")
 
     try:
+        enable_NBD(session)

```

Comment:
Minor:
Should we rename this to enable_nbd_on_all_networks()?
Also function names should be lowercase according to pep8.